test/resources/oc.resource

```diff
@@ -0,0 +1,13 @@
+*** Settings ***
+Resource    common.resource
+Library     YAML.py
+
+*** Keywords ***
+
+Oc Get
+    [Arguments]  ${type}  ${namespace}  ${resource}
+
+    ${yaml_text}=  Run With Kubeconfig  oc get -n ${namespace} -o yaml ${type} ${resource}
+    ${yaml_data}=  Yaml Parse  ${yaml_text}
+
```

Comment:
Can we add the "Set Log Level    TRACE" and "Log     ${yaml_text}" keywords be added here?